greenriver/hmis-frontend
# 186395195 stale object error when submitting household assessments

## Description

[StaleObjectError when submitting household assessments
](https://www.pivotaltracker.com/story/show/186395195)

[Capybara Tests](https://github.com/greenriver/hmis-warehouse/pull/3767)

* prevent submission of assessment when there are in-flight mutations/dirty forms
* indicate a save is in progress on the form-action bar
* pauses navigation to next assessment tab while save completes
* add some missing accessibility attrs to loading indicators

## Testing
* create a multi-member household and start an intake 
* New function: clicking Save / Submit Assessment on the bottom bar should show a blue progress bar while the form is loading

* go to the last assessment in the household
* make a change the assessment and quickly hit 'next' to go to the "complete entry" tab. You may need to throttle your network connection in chrome
* New function: loading spinner is shown on the page if the previous save is still in progress. Once the saves are complete, the summary page is visible as usual

## Type of change
[//]: # 'remove options that are not relevant'


## Checklist before requesting review
